setup.py

```diff
@@ -12,13 +12,27 @@
 if sys.version_info < (3, 5, 3):
     raise RuntimeError("aiohttp 3.x requires Python 3.5.3+")
 
+here = pathlib.Path(__file__).parent
 
 try:
     from Cython.Build import cythonize
     USE_CYTHON = True
 except ImportError:
     USE_CYTHON = False
 
+if (here / '.git').exists() and not USE_CYTHON:
+    print("Install cython when building from git clone")
+    print("pip install cython")
+    sys.exit(1)
+
+
+if (here / '.git').exists() and not (here / 'vendor/http-parser/README.md'):
+    print("Install submodules when building from git clone")
+    print("git submodule init")
+    print("git submodule update")
```

Comment:
It's simpler to do `git submodule update --init`.

Reply:
Also, how about to do that automagically if condition is met? Since we know which commands are need to run. Suddenly, it wouldn't be useful for Cython - setup_requires can't handle wheels /:

Reply:
It is for dev module only.
I prefer don't hide git commands from user here.
There is a rare possibility to have no `git` command in `PATH`, better to give a hint only

Reply:
Ok, that's fine for me.